patches/@ecl/twig-component-site-footer+3.13.0.patch

```diff
@@ -1,30 +0,0 @@
-diff --git a/node_modules/@ecl/twig-component-site-footer/site-footer.html.twig b/node_modules/@ecl/twig-component-site-footer/site-footer.html.twig
-index 761c4da..c6f58fd 100644
---- a/node_modules/@ecl/twig-component-site-footer/site-footer.html.twig
-+++ b/node_modules/@ecl/twig-component-site-footer/site-footer.html.twig
-@@ -6,6 +6,8 @@
-     - "section_class_name" (string) (default: '')
-     - "logo" (EU only)
-       - "alt" (string) alt attribute for the logo link
-+      - "aria_label" (string) aria-label attribute for the logo link
```

Comment:
We need this patch back in order to use the aria_label and title parameters that we're passing in our oe-corporate-blocks footer templates.